<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensors Table</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revert_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>341</width>
     <height>381</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>34</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>171</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>31</number>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="sensor_type_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sensor Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sensor_type_combo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Gyrocompass</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lag</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SGS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SRNS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IMU</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sensor_name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sensor Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensor_name_line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>65</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
